BP/history/characters/sami.txt

```diff
@@ -2,7 +2,7 @@
 	name="Jurva"
 	dynasty=100386
 	religion="finnish_pagan"
-	culture="lappish"
+	culture="sami"
```

Comment:
Same as above, why the name change?

Reply:
In vanilla, they changed it from Lappish CK2 to Estonian CK3 so I could've gone either way and just went with vanilla for best compatibility like above.

Reply:
Ok, fair enough on this one